paystand/paystand-magento2
# Checkout v3

## Summary
Updates the PayStand Magento 2 plugin to use PayStand Checkout v3 and fixes a number of problems related to various Magento 2 version updates.

## Jira
- [x] https://paystand.atlassian.net/browse/PPL-89
  **Magento 2 plug-in throws an error when installing on 2.1.1**
- [x] https://paystand.atlassian.net/browse/PPL-90
  **Magento 2 plug-in does not show on checkout when properly installed on Magento 2.1.1**
- [x] https://paystand.atlassian.net/browse/PPL-91
  **Magento 2 plug-in does not complete checkout when a successful payment is submitted**
- [x] https://paystand.atlassian.net/browse/PPL-92
  **Magento 2: fill out user and billing details**

## QA Notes
1. Please start with a clean Magento 2.1.1 or higher install
2. Install Magento 2 sample data so that you can test a checkout experience: http://devdocs.magento.com/guides/v2.0/install-gde/install/sample-data-before-composer.html
3. Follow the installation instructions in the README (https://github.com/steegi/paystand-magento2/blob/master/README.md), EXCEPT for these differences
3.1 Install from steegi fork of the repo
3.2 Explicitly install version 3.0.0
> `composer config repositories.paystand-magento2 git https://github.com/steegi/paystand-magento2.git`
> `composer require PayStand/PayStandMagento 3.0.0`
> `composer update`
> `php bin/magento setup:upgrade`
4. For any none user space LAMP stack, make sure to fix file permissions as they are the most common problem after installing.
5. Use a clean Sandbox customer for your tests make sure to follow the instructions on the Magento 2 integrations page
